### PR TITLE
log-backup: update version check for pitr

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11294, https://github.com/pingcap/docs-cn/pull/11315
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Leavrth — APPROVED** [review]

File: br/point-in-time-recovery.md [hunk 1]
```diff
@@ -91,3 +91,12 @@ Testing scenario 2 (on-premises):
 - During the backup process, do not exchange partition. For details, refer to [Executing the Exchange Partition DDL during PITR recovery](/br/pitr-troubleshoot.md#what-should-i-do-if-an-error-occurs-when-executing-the-exchange-partition-ddl-during-pitr-log-restoration).
 - Do not restore the log backup data of a certain time period repeatedly. If you restore the log backup data of a range `[t1=10, t2=20)` repeatedly, the restored data might be inconsistent.
 - For other known limitations, refer to [PITR Known Issues](/br/pitr-known-issues.md).
+
+### Version compatibility check
+
+In v6.3.0, backup files generated after PITR are compressed in a new manner and small files are merged before being stored (to solve problems caused by too many small files). However, this causes forward compatibility issues. See the following table:
```
Comment (TomShawn) [inline br/point-in-time-recovery.md:97]:
```suggestion
In v6.3.0, backup files generated after PITR are compressed in a new method. Small files are merged before being stored (to solve problems caused by too many small files). However, this causes backward compatibility issues. See the following table:
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: br/br-log-command-line.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Before performing log backup, you need to install Backup & Restore (BR). You can
 
 ### Enable log backup
 
-Before you use log backup, set [`log-backup.enable`](/tikv-configuration-file.md#enable-new-in-v620) in the TiKV configuration file to `true`. For the method to modify configuration, refer to [Modify the configuration](/maintain-tidb-using-tiup.md#modify-the-configuration).
+Before you use log backup, ensure that [`log-backup.enable`](/tikv-configuration-file.md#enable-new-in-v620) in the TiKV configuration file is set to the default value `true`. For the method to modify configuration, refer to [Modify the configuration](/maintain-tidb-using-tiup.md#modify-the-configuration).
```
Comment (TomShawn) [inline br/br-log-command-line.md:21]:
```suggestion
Before you use log backup, ensure that [`log-backup.enable`](/tikv-configuration-file.md#enable-new-in-v620) in the TiKV configuration file is in its default value `true`. For the method to modify configuration, refer to [Modify the configuration](/maintain-tidb-using-tiup.md#modify-the-configuration).
```

File: br/pitr-usage.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ Install or upgrade BR using TiUP:
 
 ## Enable log backup
 
-Before you use log backup, set [`log-backup.enable`](/tikv-configuration-file.md#enable-new-in-v620) in the TiKV configuration file to `true`. For the method to modify configuration, refer to [Modify the configuration](/maintain-tidb-using-tiup.md#modify-the-configuration).
+Before you use log backup, ensure that [`log-backup.enable`](/tikv-configuration-file.md#enable-new-in-v620) in the TiKV configuration file is set to the default value `true`. For the method to modify configuration, refer to [Modify the configuration](/maintain-tidb-using-tiup.md#modify-the-configuration).
```
Comment (TomShawn) [inline br/pitr-usage.md:48]:
```suggestion
Before you use log backup, ensure that [`log-backup.enable`](/tikv-configuration-file.md#enable-new-in-v620) in the TiKV configuration file is in its default value `true`. For the method to modify configuration, refer to [Modify the configuration](/maintain-tidb-using-tiup.md#modify-the-configuration).
```

Comment (shichun-0415) [inline br/pitr-usage.md:48]:
```suggestion
Before you use log backup, ensure that [`log-backup.enable`](/tikv-configuration-file.md#enable-new-in-v620) in the TiKV configuration file is in its default value `true`. For the method to modify configuration, refer to [Modify the configuration](/maintain-tidb-using-tiup.md#modify-the-configuration).
```

File: br/point-in-time-recovery.md [hunk 4]
```diff
@@ -91,3 +91,12 @@ Testing scenario 2 (on-premises):
 - During the backup process, do not exchange partition. For details, refer to [Executing the Exchange Partition DDL during PITR recovery](/br/pitr-troubleshoot.md#what-should-i-do-if-an-error-occurs-when-executing-the-exchange-partition-ddl-during-pitr-log-restoration).
 - Do not restore the log backup data of a certain time period repeatedly. If you restore the log backup data of a range `[t1=10, t2=20)` repeatedly, the restored data might be inconsistent.
 - For other known limitations, refer to [PITR Known Issues](/br/pitr-known-issues.md).
+
+### Version compatibility check
+
+In v6.3.0, backup files generated after PITR are compressed in a new method. Small files are merged before being stored (to solve problems caused by too many small files). However, as a result, TiDB clusters of the earlier version are not compatible with backup data generated in v6.3 clusters. See the following table:
```
Comment (TomShawn) [inline br/point-in-time-recovery.md:97]:
```suggestion
In v6.3.0, backup files generated after PITR are compressed in a new method. Small files are merged before being stored (to solve problems caused by too many small files). However, TiDB clusters of the earlier version are not compatible with backup data generated in v6.3 clusters. See the following table:
```

**Review by Leavrth — APPROVED** [review]

**Pull request merged**